sonar-python-plugin/src/main/java/org/sonar/plugins/python/ruff/RuffRulesDefinition.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * SonarQube Python Plugin
+ * Copyright (C) 2011-2023 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonar.plugins.python.ruff;
+
+import org.sonar.api.server.rule.RulesDefinition;
+import org.sonar.plugins.python.Python;
+import org.sonarsource.analyzer.commons.ExternalRuleLoader;
+
+public class RuffRulesDefinition implements RulesDefinition {
+
+  private static final String RULES_JSON = "org/sonar/plugins/python/ruff/rules.json";
+
+  private final ExternalRuleLoader RULE_LOADER = new ExternalRuleLoader(RuffSensor.LINTER_KEY, RuffSensor.LINTER_NAME, RULES_JSON, Python.KEY);
```

Comment:
There is code smell on this line because since it is not static and should be named in regular fields Java naming strategy.